jenkins/artifacts/jenkinsfile

```diff
@@ -216,6 +218,46 @@ def getBranchName(gitBranchName, paramBranchName) {
     return paramBranchName
 }
 
+def void sendNotification(def commitId, def status, def color) {
+    def prLink = ""
+    def number = getPRNumber()
+    if (number !=null) {
+        office365ConnectorSend (
+          status: status,
+          webhookUrl: "${TEAM_HOOK}",
+          color: color,
+          message: "<b> <font color='${color}'> PR : <a href='https://github.com/NetApp/harvest/pull/${number}'>https://github.com/NetApp/harvest/pull/${number}</a><br> Job: ${JOB_NAME} - ${BUILD_DISPLAY_NAME}</font></b>")
+    } else {
+        office365ConnectorSend (
+            status: status,
+            webhookUrl: "${TEAM_HOOK}",
+            color: color,
+            message: "<b> <font color='${color}'> Job: ${JOB_NAME} - ${BUILD_DISPLAY_NAME}</font></b>")
+    }
+}
+
+def  getPRNumber() {
```

Comment:
instead of searching all pull requests and matching by branch name, can we include the pull request number during the poll that sets the BRANCH_NAME?